projects/mercury/src/layout/UserMenu.js

```diff
@@ -91,19 +99,31 @@ const UserMenu = ({classes}) => {
                         <Paper>
                             <ClickAwayListener onClickAway={handleClose}>
                                 <MenuList className={classes.menu}>
-                                    <MenuItem className={classes.userMenu}>
-                                        <ListItemText
-                                            primary={currentUser.username}
-                                            secondary={currentUser.email}
-                                            style={{cursor: 'default'}}
-                                        />
-                                    </MenuItem>
-                                    <Divider />
-                                    <MenuItem onClick={handleLogout}>
-                                        <ListItemIcon>
-                                            <ExitToAppIcon />
-                                        </ListItemIcon>
-                                        <ListItemText primary="Logout" />
+                                    <MenuItem
+                                        className={classes.userMenu}
+                                        disableFocusOnHover
```

Comment:
There is a js warning in the console because of this:

> Warning: React does not recognize the `disableFocusOnHover` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `disablefocusonhover` instead.

Reply:
good catch, will change